train_keras.py

```diff
@@ -367,7 +385,7 @@ def evaluate(model_root, model, images, image_size, num_batches, seed):
     parser.add_argument(
         '--tensorflow_logs',
         type=str,
-        default='./tf_logs/',
+        default='./outputs/tf_logs/',
         help='Path to output tensorflow logs. Defaults to ./tf_logs/.')
```

Comment:
Change the defaults, mention that `./outputs` is root for AzureML to pay attention to logs.